action.yml

```diff
@@ -260,7 +265,7 @@ runs:
         commit_user_name: "${{ inputs.commit_user_name }}"
         commit_user_email: "${{ inputs.commit_user_email }}"
         commit_author: "${{ inputs.commit_author }}"
-        file_pattern: '${{ steps.banner.outputs.file }} ${{ steps.readme.outputs.file }}'
+        file_pattern: '${{ steps.banner.outputs.file }},${{ steps.readme.outputs.file }}'
```

Comment:
Looks like this action does use separate files by spaces. https://github.com/stefanzweifel/git-auto-commit-action

Reply:
Done